VMware Refactor (#2540)
spectrocloud/librarium: docs/docs-content/clusters/data-center/vmware/architecture.md

---
sidebar_label: "Architecture"
title: "Architecture"
description: "Learn about the architecture used to support VMware clusters through Palette."
hide_table_of_contents: false
sidebar_position: 10
tags: ["data center", "vmware", "architecture"]
---

## Overview

> The following are some architectural highlights of Kubernetes clusters provisioned by Palette on VMware:

Author: Great feedback

- Kubernetes nodes can be distributed across multiple-compute clusters, which serve as distinct fault domains.

- Support for static IP addresses, as well as DHCP. If your are using Dynamic Host Configuration Protocol (DHCP),
  Dynamic DNS is required.

- IP address pool management for assigning blocks of IPs dedicated to clusters or projects.

- A Private Cloud Gateway (PCG) that you set up within the environment to communicate with the Palette management
  platform and VMware vCenter installed in the private data center.

  The PCG facilitates communication between Palette and your infrastructure environment. The PCG is necessary in
  environments where Palette does not have direct network access. Many infrastructure environments are placed in a
  private network that blocks connections originating externally. The PCG connects to Palette, and acts as an endpoint,
  allowing you to target the environment when deploying clusters in Palette.

  ![vmware_arch_oct_2020.webp](/clusters_vmware_architecture_arch-overview.webp)

> You can learn more about the [PCG Architecture](../../pcg/architecture.md) section to learn more about the PCG

Reviewer: ```suggestion
You can learn more in the [PCG Architecture](../../pcg/architecture.md) section.
```

> architecture.

Reviewer: ```suggestion

```
